ios/lib/ReactNativeCameraKit/CKCamera.m

```diff
@@ -95,13 +96,21 @@ @interface CKCamera () <AVCaptureFileOutputRecordingDelegate, AVCaptureMetadataO
 // frame for Scanner
 @property (nonatomic, strong) NSDictionary *scannerOptions;
 @property (nonatomic) BOOL showFrame;
-@property (nonatomic) UIView *greenScanner;
+@property (nonatomic) UIView *laserView;
+@property (nonatomic) UIColor *laserViewColor;
+
+@property (nonatomic) CGFloat frameWidth;
+@property (nonatomic) CGFloat frameHeight;
+@property (nonatomic) CGFloat frameLeft;
+@property (nonatomic) CGFloat frameTop;
+@property (nonatomic) UIColor *frameColor;
+
```

Comment:
I prefer to do it in a designated object which will store all the frame props instead of dirt the camera screen component, same as `flashImages`